daq_utils.py

```diff
@@ -27,6 +27,11 @@
 soft_motor_list = []
 global screenYCenterPixelsLowMagOffset
 screenYCenterPixelsLowMagOffset = 58
+# Constants for use with C2C
+global CAMERA_ANGLE_BEAM,CAMERA_ANGLE_ABOVE, CAMERA_ANGLE_BELOW
```

Comment:
if you think about it, there are several different ways to look at these angles - looking along the beam or upstream from the detector, for CAMERA_ANGLE_BEAM. then for the ABOVE/BELOW, you can imagine top is either downstream or upstream (downstream-> away from ring). you should clarify this.

Reply:
I had thought about those other angles, but I decided to just focus on 3 simple ones to make sure it was all in place. I will expand the existing comments to clarify the specific orientation of each of these angles. We can add additional angles easily if you need them.

Reply:
ok - Kevin just verified that your description sounds correct. resolving